src/test/java/com/epam/atm/selenium/BaseTest.java

```diff
@@ -1,36 +1,23 @@
 package com.epam.atm.selenium;
 
-import org.openqa.selenium.WebDriver;
+import core.LocalCreator;
+import core.RunLocationCreator;
+import core.WebDriverSingleton;
 import org.testng.annotations.AfterClass;
 import org.testng.annotations.BeforeClass;
 
-import java.util.concurrent.TimeUnit;
-
 public abstract class BaseTest {
-    protected WebDriver driver;
-    protected String PROPERTY_CHROME = "webdriver.chrome.driver";
-    protected String PATH_TO_CHROME_DRIVER = "C:\\data\\tools\\chromedriver.exe";
-
-    protected abstract void getWebDriver();
-
-    public void setWebDriverSystemProperty() {
-        System.setProperty(PROPERTY_CHROME, PATH_TO_CHROME_DRIVER);
-    }
+    private RunLocationCreator runLocationCreator;
 
     @BeforeClass(description = "Start browser")
     public void startBrowser() {
-        getWebDriver();
-        driver.manage().window().maximize();
-        System.out.println("Maximize window");
-        driver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);
-        System.out.println("Add Implicit wait");
+        runLocationCreator = new LocalCreator();
+        WebDriverSingleton.selectDriverType(runLocationCreator);
     }
 
     @AfterClass(description = "Close browser")
     public void closeBrowser() {
-        driver.quit();
+        WebDriverSingleton.kill();
         System.out.println("Browser was closed");
```

Comment:
now u have wrapper for web driver and can move this messages to Custom web driver

Reply:
Please look through other logging and move all that could be moved

Reply:
Done